cache/src/main/java/com/example/expenselogger/cache/di/CacheModule.kt

```diff
@@ -23,3 +26,9 @@ public object CacheModule {
         return expenseDatabase.expenseDao
     }
 }
+
+@[Module InstallIn(SingletonComponent::class)]
+internal interface CacheModuleBinding {
```

Comment:
You can use one module for this. Add a companion object inside the interface, and put the other methods from the object inside

Reply:
Ohh. I didn't think of that. Thank you!